Bottom: 6fe1a6965e4dc1778c34bb6aa3c773491607733c
Top:    c32b7acc311b4ad3de023cac19244ee526bc1587
Author: Jason Wang <jasowang@redhat.com>
Date:   2015-06-25 16:57:13 +0800

Refresh of net-introduce-skb_get_swhash

---

diff --git a/include/linux/skbuff.h b/include/linux/skbuff.h
index 3380c5b..c05c8ac 100644
--- a/include/linux/skbuff.h
+++ b/include/linux/skbuff.h
@@ -926,7 +926,7 @@ static inline __u32 skb_get_hash(struct sk_buff *skb)
 	return skb->hash;
 }
 
-static inline __u32 skb_get_swhash(struct sk_buff *skb)
+static inline __u32 skb_get_sw_hash(struct sk_buff *skb)
 {
 	if (!skb->sw_hash)
 		__skb_get_hash(skb);
